tests/tfm/tfm_psa_test/prj.conf

```diff
@@ -16,7 +16,13 @@ CONFIG_TFM_ISOLATION_LEVEL=2
 # RNG
 CONFIG_PSA_WANT_GENERATE_RANDOM=y
 
-# AES Ciphers
+# Keys
+CONFIG_PSA_WANT_KEY_TYPE_AES=y
+CONFIG_PSA_WANT_KEY_TYPE_CHACHA20=y
+CONFIG_PSA_WANT_KEY_TYPE_ECC_KEY_PAIR=y
+CONFIG_PSA_WANT_KEY_TYPE_RSA_KEY_PAIR=y
```

Comment:
Should we just enable MBEDTLS_CIPHER_ALL_ENABLED maybe?
Or does it have some unwanted side-effects?

Maybe that would require less maintenance here.

Reply:
same for regression test

Reply:
These are defined as part of Zephyrs TLS configuration, I don't want to mix in those here.

I don't see this as a maintenance issue, but we can add PSA_WANT_ALL configurations in the future if we need.